database/storeDb.js

```diff
@@ -0,0 +1,127 @@
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/3000');
+
+
+var db = mongoose.connection;
+
+db.on('error', console.error.bind(console, 'connection error: '));
+db.once('open', function(){
+  console.log('mongo is connected');
+});
+
+var inventorySchema = new mongoose.Schema({
+  productId: Number, 
+  price: Number,
+  description: String, 
+  overview: Array,
+  options: Schema.Types.Mixed, //Let's me create an obj
```

Comment:
This will work but you will need to take care when generating your database seed.  Your code will need to rely on a certain format to work properly.

You can explicitly tell mongo you will have a nested format:

This is just an example:

options: [{
  key: Array
}]

Don't quote me on this specific implementation, but there is probably a safer way to define your schema so your code can always rely on the structure of the data.